modules/non_summary_pipeline.py

```diff
@@ -435,15 +435,19 @@ def from_json(file_name, element):
     """Returns an object from the JSON representation."""
 
     try:
-        return [(file_name, json.loads(element))]
+        return file_name, json.loads(element)
     except Exception as e:
         logging.error('Unable to parse file %s into JSON object "%s...": %s' % (file_name, element[:50], e))
-        return [(None)]
+        return None
 
 
 def add_date_and_client(element):
     """Adds `date` and `client` attributes to facilitate BigQuery table routing"""
 
+    if element is None:
+        logging.error('Element is emoty, skipping adding date and time')
+        return None
```

Comment:
Are you sure the next stage of the pipeline can handle None input? I think FlatMapTuple is still the right way to go so that we prune bad HARs out as early as possible.

Reply:
No I'm not sure. But I also thought it was possible that no valid files would be found (e.g. if running a small test on just corrupted files). So figure it was better to handle None here. At which point don't really need `FlatMapTuple`.